cabal-install/cabal-install.cabal

```diff
@@ -243,6 +243,7 @@ library
         zlib       >= 0.5.3    && < 0.8,
         hackage-security >= 0.6.2.0 && < 0.7,
         text       >= 1.2.3    && < 1.3 || >= 2.0 && < 2.2,
+        transformers >= 0.2    && <0.7,
```

Comment:
Anyone know if this will complicate bootstrapping?

Reply:
Should be fine, `Cabal-syntax`, `Cabal`, `cabal-install-solver`, `Cabal-hooks`, and `cabal-bootstrap-gen` already depend on `transformers`.